### DISCUSSION TITLE
Testimonials anyone!?

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [smartcontractkit/full-blockchain-solidity-course-py#1300] [Announcements]
Hi all!

I'm working on the hardhat video now, and would love some testimonials from people who took this course and now work in Web3, started a company, or just learnt a TON!

Would love to feature your snippets at the start of the hardhat video.

Let me know! 

We are almost at 2M!!!

**Comment by cromewar** [comment]
Well it's really hard for me to describe what this course meant to me, it just changed my life. I started just out of curiosity trying to find some new cool technology to learn but I was not expecting where this lead me.
Taking  this course allowed me to participate on big scale hackathons and meet amazing people around the world, the community is so charming and helpful, way better than any other community I've ever seen before.
Thanks to this course I've won a price on Moralis Hackathon, I'm working full time as a Web3 consultant for different projects and also I transforming my personal brand into a whole company.

I have to say a big thank to you @PatrickAlphaC for your amazing work and support on all my journey as a web3 developer, I just know as certainty that the Hardhat version is going to be even better and more incredible people will join Web3.

Cheers.

**Reply by PatrickAlphaC** [reply]
You've been absolutely amazing, and it's exactly people like you I make this for. Someone who is curious to learn, and then just takes everything they've learned to make a better life for themself and those around them.

I'm so excited we've been able to work together, and looking forward to continuing to do so. You're just getting warmed up ;)

**Comment by n4n0b1t3** [comment]
When I started learning web development in 1996, we only had (fricking outdated) books. Since these were sold on shelves in real bookstores, it was important for marketing reasons that the book had a very large spine. At that time there were editors whose only job was to artificially inflate the content of books so that they became thick. A book at that time had the price of $60 USD converted to today's money. Most of the time only 30% of these books were usable, the rest consisted of repetitions and there were soooo many errors. Anyway, I learned and learned, and at one point in my journey, I connected the first time a database to my application. It was this moment that everything clicked into place and I slowly realized the power of the internet.

I had a similar revelation when I created this simplistic OT token and, [following this part of the video](https://youtu.be/M576WGiDBdQ?t=30828), added it to my wallet. This course is absolutely amazing because it is at times challenging in a way that helps me to better understand the subject. The repository and forum are invaluable addition. I am incredibly grateful when I think about the wonderful tools available to one today to learn such a new and still rapidly evolving technology. @PatrickAlphaC, you put more time, effort, and thought into this course than any paid courses I am aware of.

**Reply by PatrickAlphaC** [reply]
Thank you so much for your kind words! Glad to hear that adding the token to your metamask was one of the defining moments of your web3 career!

Hopefully, this gave you the knowledge to make the jump from 1996 web dev to web3? ;)

**Comment by Noveleader** [comment]
Well, I am still learning and I am on lesson 4 but I am so sure that this course will provide me a great outcome and help me start my own company in this tech (for which I am learning it right now). Course benefits me so much learning about web3 and its components. First 1 hour describes how actually blockchain work and I love that part a lot cause it gives a idea even to newbie. I am so grateful that I found this course and got a chance to work on this tech.

**Reply by PatrickAlphaC** [reply]
Awesome, thanks for sharing!!

**Comment by mattheweller** [comment]
Gm ser. The only token I can offer you is my 2¢. This masterpiece of a tutorial, and really all your froggin' awesome content, has changed my life. The knowledge here has taken me to meet ups, a sane number of Discord servers, a DAO and two hackathons and we're just getting started BAEBEE!! 

I'm still on the road to that sweet sweet Web3 dev job but I've never felt so supported and excited to be a part of a growing community  of smart and thoughtful humans like  @PatrickAlphaC who are showing us HOW we're gonna make it.

**Reply by PatrickAlphaC** [reply]
LETS GOOOOOO!!!

Thank you so much for sharing, and good luck on your journey!!

**Comment by JoshDataGuy** [comment]
Just started learning Blockchain this month's. This course is so good. Though am new in this field I must confess for the first time am in love with something entirely different from civil engineering which I cherish so much. Thank you for this course

**Reply by Noveleader** [reply]
Great, glad you're here
